src/test/kotlin/ObserverCookieValueTest.kt

```diff
@@ -0,0 +1,14 @@
+import core.api.crm.controller.CrmController
+import core.holder.dynamicContext.DynamicContextHolder
+import org.junit.jupiter.api.Assertions
+import org.junit.jupiter.api.Test
+
+class ObserverCookieValueTest : BaseTest() {
+
+  @Test
+  fun `Add observer to validate AuthUser in a dynamic context and verify AuthUser cookie is not empty`() {
```

Comment:
Verify that AuthUser cookie value set in SessionContext when make CRM authorisation service call

Reply:
fix